index.bs

```diff
@@ -706,35 +706,23 @@ occurred on the [=remote end=].
    are run when multiple events are enabled at once, with lower integers
    indicating steps that run earlier.
 
-A [=BiDi session=] has a <dfn export for=event>global event set</dfn>
-which is a [=/set=] containing the event names for events that are enabled for all
-navigables. This initially contains the [=event name=] for events that
-are <dfn export for=event>in the default event set</dfn>.
-
-A [=BiDi session=] has a <dfn export for=event>navigable event map</dfn>,
-which is a [=/map=] with [=/top-level traversable=] keys and values
-that are a [=/set=] of [=event name=]s for events that are enabled in the given
-navigable.
-
-<div algorithm>
-
-To obtain a list of <dfn>event enabled navigables</dfn> given
-|session| and |event name|:
-
-1. Let |navigables| be an empty [=/set=].
+A [=BiDi session=] has <dfn for=event>subscriptions</dfn> which is a [=/list=] of [=subscriptions=].
 
-1. For each |navigable| → |events| of |session|'s [=navigable event map=]:
+A [=BiDi session=] has <dfn for=event>internally removed subscription ids</dfn>
```

Comment:
Given the later comments, I suggest replacing this with `A [=BiDi session=] has a <dfn for=event>known subscription ids=] which is a [=/set=] of all [=subscription/subscription ids=] that have been issued to the
[=local end=] but which have not yet been unsubscribed`. Then you unconditionally append to this when creating a subscription, and remove from it when unsubscribing by id, and get to remove a lot of logic in other places.